src/services/parser.test.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @jest-environment node
+ */
+
+import { ISelectorId } from '@/types/common';
+
+import Parse from './parser';
+
+describe('Parse:', () => {
+  const selectorIds: ISelectorId = {
+    adLinkElementAttribute: '.sVXRqc',
+    adTopParent: '#tvcap',
+    adBottomParent: '#bottomads',
+    adWords: '.uEierd',
+    nonAdWords: '.MjjYud',
+    linkElement: 'a',
+    linkElementAttribute: 'href',
+    nonAdParent: '#search',
+  };
```

Comment:
Also minor comment to add a line break

```suggestion
  };
  
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/57/commits/b22cce6b892a7ece9198950d0dbfa77466778405